### PR TITLE
CYBL-979: Use combine properties for merge node properties + runtime properties + kwargs

### PR DESCRIPTION
**Pull request by 0lvin**

### CODE REVIEW

**Review by ahmadiesa-abu — APPROVED** [review]

File: plugin.yaml [hunk 1]
```diff
@@ -71,6 +73,13 @@ node_types:
...
+            cloudify.interfaces.lifecycle:
+                create:
+                    implementation: vcloud.vcloud_network_plugin.port.create
+                    inputs: {}
+                delete:
+                    implementation: vcloud.vcloud_network_plugin.port.delete
+                    inputs: {}
             cloudify.interfaces.validation:
                 creation:
                     implementation: vcloud.vcloud_network_plugin.port.creation_validation
```
Comment (EarthmanT) [inline plugin.yaml:85]: We can actually remove `cloudify.interfaces.validation`.

File: vcloud_network_plugin/floatingip.py [hunk 2]
```diff
@@ -101,7 +109,7 @@ def _floatingip_operation(operation, vca_client, ctx):
         nat_operation = _add_nat_rule
     elif operation == DELETE:
         if not public_ip:
-            ctx.logger.info("Can't get external IP".format(public_ip))
+            ctx.logger.info("Can't get external IP {0}".format(public_ip))
```
Comment (EarthmanT) [inline vcloud_network_plugin/floatingip.py:112]: Wow. How did this never cause problems?

File: vcloud_network_plugin/keypair.py [hunk 3]
```diff
@@ -38,7 +39,12 @@ def creation_validation(**kwargs):
         check availability of path used in field private_key_path of
         node properties
     """
-    key_path = ctx.node.properties.get(PRIVATE_KEY, {}).get(PATH)
+    # combine properties
+    obj = combine_properties(ctx, kwargs=kwargs,
```
Comment (EarthmanT) [inline vcloud_network_plugin/keypair.py:43]: Is there no need for `get_manditory` here?

File: vcloud_network_plugin/floatingip.py (hunk 2 above)
Comment (0lvin) [reply vcloud_network_plugin/floatingip.py:112]: Look as no one looked to exact error message.

File: plugin.yaml (hunk 1 above)
Comment (0lvin) [reply plugin.yaml:85]: I have moved validation code to create in simple cases. #122

File: vcloud_network_plugin/keypair.py (hunk 3 above)
Comment (0lvin) [reply vcloud_network_plugin/keypair.py:43]: Looks as initially it was not mandatory, i prefer to not change logic for now.

**Pull request merged**